pypose/lietensor/lietensor.py

```diff
@@ -433,8 +459,23 @@ def translation(self, input):
     def identity(cls, *size, **kwargs):
         return SE3_type.Log(SE3_type.identity(*size, **kwargs))
 
-    def randn(self, *size, sigma=1, requires_grad=False, **kwargs):
-        data = super().randn(*size, sigma=sigma, **kwargs).detach()
+    def randn(self, *size:_size, sigma:_size_any_t=1.0, requires_grad=False, **kwargs):
+        #  convert different types of inputs to SE3 sigma
+        if not isinstance(sigma, collections.abc.Iterable):
+            sigma = _quadruple(sigma)
+        elif len(sigma)==2:
+            rotation_sigma = _single(sigma[-1])
+            transation_sigma = _triple(sigma[0])
+            sigma = transation_sigma+rotation_sigma
+        else:
+            assert len(sigma)==4
+        size = to_tuple(size)
+        rotation_data = so3_type.randn(*size, sigma=sigma[-1], **kwargs).detach().tensor()
+        transation_x = sigma[0] * torch.randn(*(size + torch.Size([1])), **kwargs)
+        transation_y = sigma[1] * torch.randn(*(size + torch.Size([1])), **kwargs)
+        transation_z = sigma[2] * torch.randn(*(size + torch.Size([1])), **kwargs)
+        transation_data = torch.cat([transation_x, transation_y, transation_z], dim=-1)
```

Comment:
use
```
sigma = torch.Tensor([sigma[0], sigma[1], sigma[2]], **kwargs)
translation = sigma * torch.randn(*(size + torch.Size([3])), **kwargs)
```
instead

Reply:
BTW, it's `translation` not `transation`

Reply:
Should use torch.tensor, not torch.Tensor
Done